lib/utils/util.js

```diff
@@ -14,6 +14,26 @@ util.copyObject = function(source, dest) {
     }
 };
 
+util.modifyArgs = (args) => {
```

Comment:
1. better to use a name which describes what is the modification that is being done.
1. lets stick in this file to functions or to arrow functions. I preferred functions because it's a more recognizable syntax, and it doesn't hint that the function is dependent on a `this` context - the main reason for using an arrow function.

Reply:
1. What name do you propose? I have no better ideas.
2. The other function in the file throws a linter warning for an unexpected unnamed function which is why I went with a different approach. So we can name both of them, but I prefer the arrow functions because they are more concise.

Reply:
Some function name ideas: "argsToOptions" , "argsToConfig" , "unflattenArgs" , "structureArgs"

Either way I'll be documenting this method (and others) shortly, it's just hard to describe in a function name.